FFXIV_Modding_Tool/Validation.cs

```diff
@@ -138,9 +144,9 @@ public bool ValidateIndexFiles()
             return keepGoing;
         }
 
-        bool PromptContinuation()
+        bool PromptContinuation(message="Would you like to continue?")
         {
-            main.PrintMessage("Continue anyway? y/N");
+            main.PrintMessage($"{message} y/N", 1);
```

Comment:
Should make also make it possible to decide if Y or N is default..? @shinnova ?

Reply:
That would be beneficial, it would make the function able to be used in more situations.